orttraining/orttraining/python/training/ortmodule/torch_cpp_extensions/_aten_op_factory.py

```diff
@@ -0,0 +1,124 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+"""Support registration of ATenOp's symbolic (PyTorch exporter overriding) and gradient definition.
```

Comment:
If docstring is used, this will be published as part of ORTModule documentation. If this is implementation detail, better to replace it with `# regular comments` as opposed to `'''docstring '''`